website/docs/maintenance/monitor-metrics.md

```diff
@@ -628,6 +628,27 @@ Some metrics might not be exposed when using other JVM implementations (e.g. IBM
       <td>The number of bytes written per second to this table.</td>
       <td>Gauge</td>
     </tr>
+    <tr>
+      <td rowspan="4">table_bucket_kv</td>
+      <td>preWriteBufferFlushPerSecond</td>
+      <td>The kv pre-write buffer flush count per second.</td>
```

Comment:
nit:
```suggestion
      <td>Th count of the flush rows in kv pre-write buffer per second.</td>
```
?
The current description always make me feel like it's kv pre-wirte buffer flush count, not the count of flushed rows in kv pre-wirte buffer